src/LibraryWindow.vala

```diff
@@ -1094,15 +1094,7 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
         }
     }
 
-    public void media_not_found(int id) {
-// XXX FIXME TODO Don't depend on ids
-#if 0
-        var not_found = new FileNotFoundDialog(library_manager, this, id);
```

Comment:
Is `FileNotFoundDialog` used somewhere else?

Reply:
Yes, it's called from `Views/ListView/Lists/MusicListView.vala`

Reply:
I couldn't get the file not found dialog to pop up because after deleting a track from my disk noise whipped it out of my library on startup, when I deleted one while in operation the app removed it from the library when I played it (in master). I think if musics going to be smarter and more automatic we should assume whatever we can't find is media that doesn't exist. However: because this is a small commit centered around removing access to class methods and doesn't introduce any regressions I give it my approval.